config/features.yml

```diff
@@ -2119,10 +2119,6 @@ features:
     actor_type: cookie_id
     description: Swaps the Search Service's for one with an updated api.gsa.gov address
     enable_in_development: true
-  remove_pciu:
-    actor_type: user
-    description: If enabled, VA Profile is used to populate contact information with PCIU backup calls
-    enable_in_development: true
```

Comment:
We have https://github.com/department-of-veterans-affairs/va.gov-team/issues/74592 set aside to delete this, but 🤷‍♀️ it makes sense to delete it here in this PR, so I'm good with this. And ☝️can be for removing it from the database in each env later.

Reply:
💯 Will delete this flipper in all environments after this has been deployed to production.